mockld/event_data.go

```diff
@@ -73,95 +38,3 @@ func (es Events) JSONString() string {
 	data, _ := json.Marshal(es)
 	return string(data)
 }
-
-func SimpleEventUser(user lduser.User) EventUser {
-	// This function can be used when we're not testing anything related to private attributes
-	return ExpectedEventUserFromUser(user, servicedef.SDKConfigEventParams{})
-}
-
-func ExpectedEventUserFromUser(user lduser.User, eventsConfig servicedef.SDKConfigEventParams) EventUser {
```

Comment:
This is replaced by `eventUserMatcher` in `server_side_events_users.go`, which uses composable matchers instead of building a JSON object. I figured it made more sense for the logic to be in that file because now it's _only_ the tests in that file that make assertions about specific user properties in events.